src/main/resources/META-INF/plugin.xml

```diff
@@ -166,13 +166,6 @@
                 <override-text place="GitMacheteToolbar"/>
             </action>
 
-            <action id="GitMachete.RefreshStatusAction"
```

Comment:
Pls check `git grep -in refresh`... some occurrences indicate usages that can still be removed (e.g. `REFRESH_STATUS` icon, `docs/features.md`, and `docs/plugins.jetbrains.com/getting_started.html`